Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -743,6 +751,30 @@ noncomputable def inclusion.fullyFaithful (n : ℕ) :
 theorem Hom.ext {n} {a b : Truncated n} (f g : a ⟶ b) :
     f.toOrderHom = g.toOrderHom → f = g := SimplexCategory.Hom.ext _ _
 
+/-- Some quick and useful attempts to prove `m ≤ n`. -/
+macro "leq_tac" : tactic =>
+  `(tactic| first | decide | assumption | apply zero_le | apply le_rfl |
+    apply Nat.le_add_left | apply Nat.le_add_right |
+    apply Nat.le_add_right_of_le; assumption |
+    apply Nat.add_le_add_right; assumption | transitivity <;> assumption)
```

Comment:
How expensive is `omega`? Is this the right balance of things to try before delegating to `omega`?